public/app/features/explore/TraceView/components/TraceTimelineViewer/SpanBarRow.tsx

```diff
@@ -492,7 +492,12 @@ export class UnthemedSpanBarRow extends React.PureComponent<SpanBarRowProps> {
                 const links = createSpanLink(span);
                 const count = countLinks(links);
                 if (links && count === 1) {
-                  const link = links.logLinks?.[0] ?? links.metricLinks?.[0] ?? links.traceLinks?.[0] ?? undefined;
+                  const link =
+                    links.logLinks?.[0] ??
+                    links.metricLinks?.[0] ??
+                    links.traceLinks?.[0] ??
+                    links.otherLinks?.[0] ??
```

Comment:
@gelicia do you think it make sense to keep the distinction? I mean apart from having some label all these types work the same right? I feel like we could just use some link label/name so they make sense but maybe we don't need a specific sections for them.

Reply:
yeah it's probably not going to be tons of them (year later there will be somebody with >100 links complaining about this :D ) but also how do we know some link from the `otherLinks` isn't going to be a log or trace link? I assume correlations can point anywhere and either we will have some "type" selection there (which not sure if that makes sense, what if link points to multiple things like both logs and metrics) or we just don't try to group them at all.

Reply:
That's true. Unless we keep some sort of datasource categorization in here by the datasource "type" we will have no way of knowing if its a logs/metrics/etc link. 

A correlations link now cannot point to a mixed datasource setup, but if a datasource contained both logs and metrics you could have that scenario.

I see what you mean about the grouping. I will take that out for this PR.